subiquity/server/apt.py

```diff
@@ -75,7 +78,7 @@ def _lowerdir_for_mnt(mnt):
 def _lowerdir_for_ovmnt(ovmnt):
     # One cannot indefinitely stack overlayfses so construct an
     # explicit list of the layers of the overlayfs.
-    return lowerdir_for(ovmnt.lowers + [ovmnt.upperdir])
+    return lowerdir_for([ovmnt.lowers] + [ovmnt.upperdir])
```

Comment:
[ovmnt.lowers] + [ovmnt.upperdir] can be written [ovmnt.lowers, ovmnt.upperdir] :)

It took some me some work to convince myself this was correct. I really should have written unit tests for this :/

Reply:
lol - Thanks!